### PR TITLE
Insertion

### PR DESCRIPTION
**Pull request by ahrenswett**

### CODE REVIEW

File: src/test/java/code401challenges/InsertionSortTest.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    public void setup() {
+
+    }
+    @Test
+    public void insertTest(){
+        int[] testarr = {3,5,2,4,1,7,6};
+        int[] answer = {1,2,3,4,5,6,7};
+        assertArrayEquals(answer,
+        InsertionSort.insertSort(testarr));
+    }
```
Comment (jeffborda) [inline src/test/java/code401challenges/InsertionSortTest.java:19]: This is a good test for "normal" case, but you definitely want to test edge cases, : array length 0, array length 1, array with duplicate numbers, array with negative numbers, etc.

File: src/main/java/code401challenges/InsertionSort.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
+package code401challenges;
+
+public class InsertionSort {
```
Comment (jeffborda) [inline src/main/java/code401challenges/InsertionSort.java:3]: I recommend making a package called "sort" where you can put your sorting algorithms.
